### ISSUE TITLE
Comments get parsed if on same line as tag

### ISSUE THREAD

**Issue by pplettner** [althonos/pronto#8]
I'm importing the obi.owl file from the OBI ontology (http://obi-ontology.org/) and after many tags they add a comment on the same line. Here's an example from line 5:

`<owl:versionIRI rdf:resource="http://purl.obolibrary.org/obo/obi/2017-02-22/obi.owl"/><!-- OBI Release 2017-02-22 -->`

This causes an error in the parser which thinks that the comment tag is a real element:
```
File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/ontology.py", line 80, in __init__
    self.parse(handle, parser)
  File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/ontology.py", line 197, in parse
    self.meta, self.terms, self.imports = p.parse(stream)
  File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/parser/owl.py", line 108, in parse
    meta, imports = cls._parse_meta(tree)
  File "/home/pplettner/venv/lib/python3.5/site-packages/pronto/parser/owl.py", line 127, in _parse_meta
    basename = elem.tag.split('}', 1)[-1]
AttributeError: 'cython_function_or_method' object has no attribute 'split'
```
Comment tags on their own lines are ignored, and if I delete all of these postfix comments in the OWL file it imports successfully. Could you fix this to ignore these comments as well?

Thanks in advance.

**Comment by althonos** [comment]
Hi, thank you very much for this bug report. This only seems to occur with `lxml` as the XML parsing library (since the standard library skips comments entirely). I attempted a fix and will hopefully push release `v0.7.2` shortly.

**Comment by althonos** [comment]
Fixed in `v0.7.3`.
